.circleci/behat-test.sh

```diff
@@ -55,11 +67,15 @@ terminus -n env:wake $TERMINUS_SITE.$TERMINUS_ENV
 # Ping wp-cli to start ssh with the app server
 terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- cli version
 
-# Run the Behat tests
-cd tests && ../vendor/bin/behat --config=behat/behat-pantheon.yml --strict "$@"
+# Verbose mode and exit on errors
+set -ex
```

Comment:
Verbose + exit on errors just before Behat. Part of the reason for this is that wp-cli will have a non-zero status code for commands like `user list --login=some_user` if that user doesn't exist.